src/foundation.cs

```diff
@@ -13493,7 +13493,6 @@ interface NSUnitTemperature : NSSecureCoding {
 	}
 
 #if !WATCH && !TVOS
-	[Mac (10,8), iOS (11,0), NoWatch, NoTV]
```

Comment:
Mmh wouldn't that remove the iOS 11 availability attribute on `TrashItem`?

Reply:
You're right, I missed that.